app/scripts/app.js

```diff
@@ -88,7 +88,21 @@ angular
       }
     };
   })
+  .factory('loggedOutInterceptor', function($q, alertService) {
+    return {
+      responseError: function (rejection) {
+        var status = rejection.status;
+        if (status === 401) {
```

Comment:
How do we distinguish between session timed out and unauthorized access to a certain resource (by example query admin rest api with customer credentials)?